band-consumer/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Simple usage with a mounted data directory:
```

Comment:
This is a good file to make an image for daemon and client. Just change gaiad -> bcd